app/assets/javascripts/guides/new.js

```diff
@@ -322,6 +365,15 @@ openFarmApp.controller('newGuideCtrl', ['$scope', '$http', '$filter',
     $scope.newGuide.crop.description = '';
   };
 
+  //Gets fired when user resets their selection.
+  $scope.clearCropSelection = function ($item, $model, $label) {
```

Comment:
'$label' is defined but never used.<br>'$model' is defined but never used.<br>'$item' is defined but never used.